lib/created_message_queue_handler.ts

```diff
@@ -125,16 +125,22 @@ export async function handleMessage(
 
   if (isMessageStorageEnabled) {
     // if the recipient wants to store the messages
-    // we add the content of the message to the message record
-    const errorOrUpdatedMessage = await messageModel.update(
+    // we add the content of the message to the blob storage for later retrieval
+    const errorOrRetrievedMessage = await messageModel.attachStoredContent(
       retrievedMessage.id,
       retrievedMessage.fiscalCode,
-      m => {
-        return { ...m, content: messageContent };
+      {
+        content: messageContent,
+        // we do not store documentdb metadata
+        fiscalCode: retrievedMessage.fiscalCode,
+        senderOrganizationId: retrievedMessage.senderOrganizationId,
+        senderUserId: retrievedMessage.senderUserId
       }
     );
 
-    if (errorOrUpdatedMessage.isLeft) {
+    winston.debug(`storingMessageContent|${JSON.stringify(retrievedMessage)}`);
```

Comment:
if you want to follow my convention, first identifier is the method name, i.e. `handleMessage`

Reply:
ok